internal/cri/server/sandbox_stats_list.go

```diff
@@ -34,20 +35,32 @@ func (c *criService) ListPodSandboxStats(
 	r *runtime.ListPodSandboxStatsRequest,
 ) (*runtime.ListPodSandboxStatsResponse, error) {
 	sandboxes := c.sandboxesForListPodSandboxStatsRequest(r)
+	stats, errs := make([]*runtime.PodSandboxStats, len(sandboxes)), make([]error, len(sandboxes))
+
+	eg := new(errgroup.Group)
+	for i, sandbox := range sandboxes {
+		i := i
+		eg.Go(func() error {
+			sandboxStats, err := c.podSandboxStats(ctx, sandbox)
+			switch {
+			case errdefs.IsUnavailable(err), errdefs.IsNotFound(err):
+				log.G(ctx).WithField("podsandboxid", sandbox.ID).Debugf("failed to get pod sandbox stats, this is likely a transient error: %v", err)
```

Comment:
nit: prefer `WithError(err)` over `%v`

Reply:
Done. But FTR `%v` was used before this PR.